### DISCUSSION TITLE
課題12についての質問

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#39] [Q&A]
https://github.com/nsc-kiwamu/traning-takeda/pull/38#issue-1310683251

↑上記のリンク先にて、質問を書かせていただいております。
よろしくお願いいたします。

**Comment by nsc-kiwamu** [comment]
・課題2
できているようでできていない。
https://github.com/nsc-kiwamu/traning-takeda/pull/38#discussion_r925607437

・課題3
https://github.com/nsc-kiwamu/traning-takeda/pull/38#discussion_r925614874

・課題4
あくまで１例ですが↓のような考えで進めてみて下さい
https://github.com/nsc-kiwamu/traning-takeda/pull/38#discussion_r925615898
https://github.com/nsc-kiwamu/traning-takeda/pull/38#discussion_r925617323
https://github.com/nsc-kiwamu/traning-takeda/pull/38#discussion_r925617812

・課題5
すみません、サンプルが悪いですね。。。
https://github.com/nsc-kiwamu/traning-takeda/pull/38/files#r925628652
